src/libYARP_dev/src/devices/ControlBoardWrapper/RPCMessagesParser.cpp

```diff
@@ -416,7 +418,8 @@ YARP_WARNING_POP
                     {
                         js[i] = lIn.get(i).asInt();
                     }
-                    *ok = rpc_iCtrlMode2->getControlModes(n_joints, js, modes);
+                    if(rpc_iCtrlMode2)
+                        *ok = rpc_iCtrlMode2->getControlModes(n_joints, js, modes);
```

Comment:
See previous comment.

Reply:
Fixed in https://github.com/robotology/yarp/commit/6b79a0c894708c3ef19992997f1ad1ba8968f39e